### ISSUE TITLE
Missing apisixpluginconfigs in apisix-ingress-controller rbac.yml

### ISSUE THREAD

**Issue by suninuni** [apache/apisix-helm-chart#210]
Missing apisixpluginconfigs in apisix-ingress-controller rbac.yml caused this error:

```
E0106 15:12:27.640166       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.22.4/tools/cache/reflector.go:167: Failed to watch *v2beta3.ApisixPluginConfig: failed to list *v2beta3.ApisixPluginConfig: apisixpluginconfigs.apisix.apache.org is forbidden: User "system:serviceaccount:kube-system:apisix-ingress-controller" cannot list resource "apisixpluginconfigs" in API group "apisix.apache.org" at the cluster scope
```

https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix-ingress-controller/templates/rbac.yaml#L137-L151

**Comment by suninuni** [comment]
@tao12345666333 FYI

**Comment by tao12345666333** [comment]
ApisixPluginConfig is not really available until the v1.4 version, so its related configuration will also be carried in the v1.4 version of the APISIX Ingress controller Helm chart. #208
